client/src/components/EventView/Header/index.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FiPrinter, FiDownload } from "react-icons/fi";
+import styled from "styled-components";
+
+import IconButton from "../../Common/IconButton";
+import Stack from "../../Common/Stack";
+import {
+  theme1,
+  fontSize64,
+  fontSize24,
+  spacing8,
+  spacing16,
+  sideMargins
+} from "../../../resources/style-constants";
+
+const Container = styled.div`
+  background-color: ${(props) => props.bgcolor || theme1};
+  padding: 0 ${sideMargins};
+`;
+
+const TitleContainer = styled.div`
+  width: 100%;
+`;
+
+const TextContainer = styled.div`
+  height: 100%;
+  width: calc(100% - ${spacing8});
+  margin-right: ${spacing16};
+  font-size: ${fontSize64};
+`;
+
+const DateContainer = styled.div`
+  font-size: ${fontSize24};
+`;
+
+const DateTextContainer = styled.div`
+  opacity: 0.5;
+  margin: 0;
+`;
+
+const ButtonBarContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+/**
+ * The header which appears at the top of the event-view page.
+ */
+const Header = ({
+  event: {
+    title,
+    date: { startDate }
+  },
+  backgroundColor,
+  onPressDownload,
+  onPressPrint,
+  ...props
+}) => {
+  const realDate = new Date(startDate);
+  return (
+    <Container {...{ bgcolor: backgroundColor, ...props }}>
+      <Stack>
+        <TitleContainer>
+            <TextContainer>{title}</TextContainer>
+        </TitleContainer>
+        <DateContainer>
+          <DateTextContainer>{realDate.toDateString()}</DateTextContainer>
```

Comment:
Josiah created a useful component for formatting dates. It's located in /Common/Date. You can add a new DateFormat to it with whatever format you want, like "MMMM D, YYYY", which would print out December 3, 2020. Let me know if you need more help with that part